### PR TITLE
DF-228: Fix 0 results showing for 'Find an archive' bucket

### PR DESCRIPTION
**Pull request by ababic**
Resolves ticket: https://national-archives.atlassian.net/browse/DF-228

### CODE REVIEW

File: etna/search/views.py [hunk 1]
```diff
@@ -161,9 +163,11 @@ def catalogue_search(request):
                 Aggregation.CATALOGUE_SOURCE,
                 Aggregation.CLOSURE,
                 Aggregation.COLLECTION,
-                Aggregation.GROUP,
                 Aggregation.LEVEL,
                 Aggregation.TOPIC,
+                # Fetching more groups so that we receive a counts
+                # for any bucket/tab options we might be showing
+                f"{Aggregation.GROUP}:30",
             ],
```
Comment (TNA-Allan) [inline etna/search/views.py:171]: Can we have it as a constant like 'AGGREGATION_SIZE' used in catalogue_search_long_filter_chooser than hardcode?

File: etna/search/views.py [hunk 2]
```diff
@@ -294,9 +298,11 @@ def website_search(request):
...
```
Comment (TNA-Allan) [inline etna/search/views.py:306]: Can we have it as a constant like 'AGGREGATION_SIZE' used in catalogue_search_long_filter_chooser than hardcode?

**Review by TNA-Allan — CHANGES_REQUESTED** [review]

**Comment by TNA-Allan** [comment]
Branch name convention does not contain ticket df-228

File: etna/search/views.py (hunk 1 above)
Comment (ababic) [reply etna/search/views.py:171]: I'm not sure that's worth it, personally? I think the comment does a good enough job of explaining what's happening - and using a constant in just this one case and not the others, where we're relying on the default (10)

**Comment by ababic** [comment]
> Branch name by convention does not contain ticket df-228

My bad - shouldn't be a problem though, as the branch will be deleted after merging.

File: etna/search/views.py (hunk 1 above)
Comment (TNA-Allan) [reply etna/search/views.py:171]: Somewhat agree, but I will leave this to your best judgment :)

**Review by TNA-Allan — APPROVED** [review]
Looks good to me

File: etna/search/views.py (hunk 1 above)
Comment (ababic) [reply etna/search/views.py:171]: In this case, I think including 30 directly is slightly clearer.

**Pull request merged**